input/pagecontent/fhirartifacts.md

```diff
@@ -0,0 +1,5 @@
+  - [Artefacts Summary](artifacts.html)
+  - [Profiles and Extensions](profiles-and-extensions.html)
+  - [Terminology](terminology.html)
+  - [Actors and Capabilities](capability-statements.html)
+  
```

Comment:
Commenting here to get an inline comment. 

[Guidance](https://build.fhir.org/ig/hl7au/au-fhir-erequesting/branches/ft_update-menus-like-base/guidance.html) page requires a few small changes:
- 'eRequesting Data for Interoperability' should be 'AU eRequesting Data for Interoperability Mappings' to match the page title
- 'Relationship with other IGs' should be 'Relationships to Other Specifications' to match the page title.